psalm.xml

```diff
@@ -195,6 +195,7 @@
         <PossiblyNullArgument>
             <errorLevel type="suppress">
                 <file name="src/Bundle/Grid/Parser/OptionsParser.php" />
+                <file name="src/Component/src/Metadata/Operation/HttpOperationInitiator.php" />
```

Comment:
Psalm is totally wrong here... I think we'll need to remove Psalm at some point.
PHPStan is better nowadays.